components/sections/settingsSection/idpSection/otherComponents/buttonGroupIdentityProviderDetails.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+  errorTypeDialog,
+  successTypeDialog,
+} from "../../../../common/dialogComponent/dialogComponent";
+import { checkIfJSONisEmpty } from "../../../../../utils/util-common/common";
```

Comment:
We can use the @ notation to avoid these relative paths.

cc: @Achintha444 

```suggestion
import { checkIfJSONisEmpty } from "@/utils/util-common/common";
```